docs/process.md

```diff
@@ -44,16 +44,37 @@ When:
 
 Requirements:
 
- * GitHub CI is green. (Currently, that includes good coverage for Linux, but
-   nothing else - we should keep trying to find resources to do better here.)
+ * [emscripten-releases build CI](https://ci.chromium.org/p/emscripten-releases/g/main/console)
+   is green on all OSes for the desired hash (where the hash is the git hash in
+   the
+   [emscripten-releases](https://chromium.googlesource.com/emscripten-releases)
+   repo, which then specifies through
+   [DEPS](https://chromium.googlesource.com/emscripten-releases/+/refs/heads/master/DEPS)
+   exactly which revisions to use in all other repos).
+ * [GitHub CI](https://github.com/emscripten-core/emscripten/branches) is green
+   on the incoming branch.
 
 How:
 
- * Ask on irc if there are any concerns.
- * The emscripten, emscripten-fastcomp, and emscripten-fastcomp-clang repos
-   should each be updated: the emscripten-version.txt file in each, and a git
-   tag (with the simple version number).
- * A tag should also be done in the binaryen repo.
+ * Open a PR for the emsdk to update
+   [emscripten-releases-tags.txt](https://github.com/emscripten-core/emsdk/blob/master/emscripten-releases-tags.txt),
+   adding the version and the hash. Updating the "latest" tag there to the new
+   release is possible, but can also be deferred if you want to do more testing
+   before users fetching "latest" get this release.
+ * Tag the emscripten repo on the emscripten commit used by that release (which
+   you can tell from the DEPS file), using something like
+   `git checkout [COMMIT]` ; `git tag [VERSION]` ; `git push --tags`.
+ * Update
```

Comment:
This means we won't have "dev" or 0 versions, right? So don't forget to update the commit message.